packages/help/src/example/test.js

```diff
@@ -1,60 +1,20 @@
-import test from 'tape-promise/tape'
+import { OrderedMap } from 'immutable'
 
-import { createHelpStore } from './store'
 import { actions as helpActions } from '../redux'
-import { searchHelpTopics } from './fetch'
-import {  OrderedMap, Map } from 'immutable'
+import { createHelpStore } from './store'
 
-test('Has initial state.', (assert) => {
+
+test('Has initial state.', () => {
   const store = createHelpStore()
   const state = store.getState()
 
-  assert.equal(state.help.helpQuery, '',
-    'Initial help query should be empty string')
-
-  assert.equal(state.help.results, OrderedMap(),
-    'Initial help results should be ordered map')
-
-  assert.end()
+  expect(state.help.helpQuery).toBe('')
+  expect(OrderedMap.isOrderedMap(state.help.results)).toBe(true)
 })
 
-test('Query can be set with an action.', (assert) => {
+test('Query can be set with an action.', () => {
   const store = createHelpStore()
   store.dispatch(helpActions.setHelpQuery('testing'))
   const state = store.getState()
-  assert.equal('testing', state.help.helpQuery)
-  assert.end()
-})
-
-test('Fetch stuff from database.', (assert) => {
```

Comment:
These tests depended on an API server and Elasticsearch instance which we don't have a test environment for.